src/models/bandwidthMessageItem.ts

```diff
@@ -42,5 +42,5 @@ export const bandwidthMessageItemSchema: Schema<BandwidthMessageItem> = object({
   segmentCount: ['segmentCount', optional(number())],
   errorCode: ['errorCode', optional(number())],
   receiveTime: ['receiveTime', optional(string())],
-  carrierName: ['carrierName', optional(string())],
+  carrierName: ['carrierName', nullable(optional(string()))],
```

Comment:
```suggestion
  carrierName: ['carrierName', nullable(optional(string()))],
```
 Supposed to be `optional(nullable(x))`?

Reply:
they're commutative in this instance, each version allows for `undefined`, `null`, and `string` as valid types